src/OpenTelemetry.Exporter.Zipkin/ZipkinExporterOptions.cs

```diff
@@ -51,5 +51,30 @@ public sealed class ZipkinExporterOptions
         /// </summary>
         public int? MaxPayloadSizeInBytes { get; set; } = DefaultMaxPayloadSizeInBytes;
 #endif
+
+        /// <summary>
+        /// Gets or sets the exporter type for Zipkin Exporter.
+        /// </summary>
+        public ZipkinExporterType ExporterType { get; set; } = ZipkinExporterType.SimpleExportProcessor;
```

Comment:
Do we want the simple one to be default?

Reply:
Lets use Batch, as thats what we think every exporter making any http/rpc call to be using to be efficient.

Reply:
Fixed this.